apps/emqx/src/emqx_limiter/src/emqx_limiter_schema.erl

```diff
@@ -130,39 +133,9 @@ fields(node_opts) ->
 fields(client_fields) ->
     client_fields(types(), #{default => #{}});
 fields(bucket_infinity) ->
-    [
-        {rate, ?HOCON(rate(), #{desc => ?DESC(rate), default => <<"infinity">>})},
-        {burst,
-            ?HOCON(capacity(), #{
-                desc => ?DESC(capacity),
```

Comment:
Strange, there's no `capacity` in the `.hocon` i18n file, but the CI checks passed.  I think it may be due to this field being hidden.  :thinking: